### DISCUSSION TITLE
Streak error

### DISCUSSION THREAD

**Discussion by Sahillather002** [cssbattle/forum#383] [Bugs reports]
I have played for 3 days daily but streak shown as 2 , actually on second day it didn't get updated when i played on 3rd day its updated to 2 thus max to 2.
![image](https://github.com/cssbattle/forum/assets/72077931/5a6fe775-b772-4e7a-a1f6-8385ad94335b)
![image](https://github.com/cssbattle/forum/assets/72077931/e6ba3e28-650b-40f6-89c3-dbaf5b6c1656)

**Comment by chinchang** [comment]
Hi @Sahillather002 
You made 11 submissions on 9th Feb and 1 submission on 10th Feb (IST). So max streak of 2 seems okay. And since no submission has been made after 10th, your current streak is 0.

**Reply by Sahillather002** [reply]
what about 8th i also solved that day also,
when i did on 9th daily streak didn't updated to 2 it remains 1 for some reason and
when i didn on 10th its updated to 2 
that's why i see something wrong with streak
